modules/azurite/testcontainers/azurite/__init__.py

```diff
@@ -102,12 +101,7 @@ def get_connection_string(self) -> str:
 
         return connection_string
 
-    def start(self) -> "AzuriteContainer":
-        super().start()
-        self._connect()
-        return self
-
     @wait_container_is_ready(OSError)
-    def _connect(self) -> None:
+    def _wait_until_ready(self) -> None:
         with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
             s.connect((self.get_container_host_ip(), int(self.get_exposed_port(next(iter(self.ports))))))
```

Comment:
Might it make sense to also pull this into utils as a `wait_for_port`? At least I had the need in my code to basically write something similar:

```py
        @wait_container_is_ready(NoSuchPortExposed)
        def _wait_for_port() -> None:
            zks.get_service_port("schemaregistry", 8085)

        _wait_for_port()
```